src/plugins/api/elements.h

```diff
@@ -168,7 +168,7 @@ class Element : public Ms::PluginAPI::ScoreElement
     API_PROPERTY_T(bool,   visible,       VISIBLE)
     /** Stacking order of this element */
     API_PROPERTY_T(int,    z,             Z)
-    API_PROPERTY(small,                   SMALL)
+    API_PROPERTY(isSmall,                 SMALL)
```

Comment:
This seems not needed and might break plugins using `small`

Reply:
It does cause an mtest failure (in 3.x):
```
52/62 Test #53: tst_scripting ....................***Failed    1.09 sec
This plugin does not support createPlatformOpenGLContext!
--- /home/runner/work/MuseScore/MuseScore/mtest/scripting/p1.log.ref	2021-08-03 11:40:32.524648773 +0000
+++ p1.log	2021-08-03 12:05:47.534379914 +0000
@@ -4,13 +4,13 @@
 found:TimeSig (24) at 0
 found:Chord (93) at 0
   beamMode:0
-  small:false
+  small:
   stemDirection:AUTO
   duration:1/4
 found:Rest (25) at 480
 found:Chord (93) at 960
   beamMode:0
-  small:false
+  small:
   stemDirection:AUTO
   duration:1/2
 found:BarLine (12) at 1920
   <diff -u /home/runner/work/MuseScore/MuseScore/mtest/scripting/p1.log.ref p1.log failed

```

Reply:
Okay, I'll undo this change! (and other plugin API related ones, if any)